scripts/exregional_run_post.sh

```diff
@@ -166,7 +166,7 @@ fi
 # Set the names of the forecast model's write-component output files.
 #
 if [ "${RUN_ENVIR}" = "nco" ]; then
-    DATAFCST=$DATAROOT/run_fcst${dot_ensmem/./_}.${share_pid}
+    DATAFCST=$DATAROOT/run_fcst_mem${ENSMEM_INDX}.${share_pid}
```

Comment:
Does the directory name also have _mem000 for non-ensemble runs now? I thought this addition affected only task names.

Reply:
Yes, it does. Without this change, the output files of the run_post are not generated. The issue is that the log file of `run_post` says that the dir/file is not found, but the task is completed successfully and generates "SUCCESS" message at the end.

Reply:
Ok thanks! I am going to have to make similar modifications in rrfs dev where `dot_ensmem` is used.
Can we modify "dot_ensmem" defintion in `job_preamble.sh` instead and keep the old way of setting DATAFCST?
I think we just have to remove " [ "${DO_ENSEMBLE}" = "TRUE" ] " in the if condition
```
if [ "${RUN_ENVIR}" = "nco" ] && [ ! -z $ENSMEM_INDX ]; then
    export dot_ensmem=".mem${ENSMEM_INDX}"
else
    export dot_ensmem=
fi
```

Reply:
But we have to be careful though, since this will also modify the names of output files.
```
${COMOUT}/${NET}.${cycle}${dot_ensmem}.prslev.f${fhr}.${POST_OUTPUT_DOMAIN_NAME}.grib2
```
Modifying `dot_ensmem` will add a ".mem000" to the name which we probably don't want.
I am not sure it is a good idea so forget that I suggested it.
Non-ensemble runs should not have been modified to have a mem000 anywayere.